demo/billing/resources/billing.proto

```diff
@@ -26,7 +26,7 @@ message Record {
   google.protobuf.Timestamp interval_end = 3;
 
   string meter = 4; // What's being measured
-  uint32 value = 5;
+  int32 value = 5;
```

Comment:
Note for the future:

Materialize doesn't support `uint`s! This didn't fail before because the `value` field was nested, and all nested fields were casted to floats. Now that we're using the correct types, it needed to be corrected from `uint32` -> `int32`.

Reply:
cc https://github.com/MaterializeInc/materialize/issues/7629